validation/longhorn/schemas/pit_schemas.md

```diff
@@ -0,0 +1,399 @@
+
+| Step Number | Action | Data | Expected Result                       |
+|------|--------|--------|----------------------------------------|
+| 1 | Create backup of all critical volumes before upgrade | | Backups completed successfully         |
+| 2 | Document current Longhorn version and settings | | Baseline configuration recorded        |
+| 3 | Navigate to "Apps" → "Installed Apps" in Rancher | | Installed applications list displays   |
+| 4 | Locate Longhorn application and click "..." → "Upgrade" | | Upgrade interface opens                |
+| 5 | Review upgrade notes and breaking changes | | Upgrade requirements verified          |
+| 6 | Select target version (e.g., v1.9.0) | | Target version selected                |
+| 7 | Review configuration and preserve custom settings | | Settings maintained through upgrade    |
+| 8 | Click "Upgrade" to begin process | | Upgrade process initiates              |
+| 9 | Monitor upgrade progress in Apps section | | Upgrade proceeds without errors        |
+| 10 | Verify all Longhorn components reach new version | | All pods updated to target version     |
+| 11 | Test volume operations post-upgrade | | Create, attach, detach operations work |
+| 12 | Verify existing volumes remain functional | | All volumes accessible and healthy     |
+
+### Kubernetes Cluster Upgrade Compatibility
+
+Test Kubernetes cluster upgrade through Rancher while maintaining Longhorn storage functionality.
+
+| Step Number | Action | Data | Expected Result                             |
+|------|--------|--------|----------------------------------------------|
+| 1 | Verify Longhorn compatibility with target K8s version | | Compatibility matrix confirms support        |
+| 2 | Create backup of critical data | | Data backup completed                        |
+| 3 | Navigate to "Cluster Management" and select cluster | | Cluster configuration page opens             |
+| 4 | Click "Edit Config" and select new Kubernetes version | | Target version selected                      |
+| 5 | Review upgrade plan and rolling update strategy | | Upgrade plan shows minimal downtime          |
+| 6 | Initiate cluster upgrade through Rancher | | Upgrade begins with control plane            |
+| 7 | Monitor control plane upgrade completion | | Control plane updated successfully           |
+| 8 | Monitor worker node upgrades (rolling updates) | | Nodes upgrade one by one                     |
+| 9 | Keep Longhorn UI open to monitor volume status | | Volumes remain accessible throughout         |
+| 10 | Verify no volume degradation during node upgrades | | All volumes maintain healthy status          |
+| 11 | Test volume operations after cluster upgrade | | Volume creation and operations work normally |
+| 12 | Verify all nodes at target Kubernetes version | | Cluster fully upgraded                       |
+
+### High-Volume Concurrent Operations
```

Comment:
I like this test. cc @rancher-max would be nice to have as a feature here to add the labels qase has for tests i.e. stress test

Reply:
Agreed! With the update from Sam that just came in, we should be able to do that now!